src/Mod/PartDesign/Gui/SketchWorkflow.cpp

```diff
@@ -387,11 +387,18 @@ class PlaneFinder
     void findDatumPlanes()
     {
         App::GeoFeatureGroupExtension *geoGroup = getGroupExtensionOfBody();
-        auto datumPlanes( appdocument->getObjectsOfType(PartDesign::Plane::getClassTypeId()) );
+        auto datumPlanes = appdocument->getObjectsOfType(PartDesign::Plane::getClassTypeId());
+        auto datumPlanes2 = appdocument->getObjectsOfType(App::Plane::getClassTypeId());
```

Comment:
datumPlanes2? That should be named better, `appDatumPlanes` would be probably fine enough, but maybe someone have better name in mind.

Reply:
I think datumPlanes2 is a ok name in this case because it does not matter than they are app or PD planes. Especially since those variables get merged right after.
So adding 'app' to the name does not appear to bring any value.

Perhaps the real problem here is that we should have a `getObjectsOfType(vector<types>)` function

Reply:
Yup, that would work well, thanks, I commented on use of std::vector instead of std::initializer_list - but conceptually it is great.